README.rst

```diff
@@ -45,5 +45,18 @@ an issue or if you need any help then feel free to ping us in gitter channel.And
 .. |gitter channel| image:: https://badges.gitter.im/Join Chat.svg
  :target: https://gitter.im/Smvdu-Algos/Lobby
  :align: middle
+ 
+ **Complexities of various Search & Sort Algos:**
+ 
+ *To get a better understanding of the best,average and worst case complexities of various search and sort algos,take a look at              the following images And to understand what is BIG O Complexity, read the following blog:
+ The Beginner's guide to Big O Complexity- https://rob-bell.net/2009/06/a-beginners-guide-to-big-o-notation/
+ 
+**Happy Coding.****GodSpeed..!!**
+ 
+=====
```

Comment:
Add a blank line after line no. 56 and 59 😉

Reply:
Okayzz Sir..